README.md

```diff
@@ -56,8 +56,8 @@ BifrostBridge::resolveRoleClassUsing(function(/* auto injection works here */):
 // Disabled role sync
 BifrostBridge::resolveRoleClassUsing(fn() => null);
 
-// If the system does not get an email, a user is retrieved using this method
-BifrostBridge::resolveUserWithoutEmailUsing(function(/* auto injection works here */, BifrostUserData $data): ?EloquentModel {
+// Override the way a user is resolved
```

Comment:
Misschien hier toevoegen dat deze functie ook het User model moet updaten als dat nodig is, en de rollen moet syncen ? En eventueel hernoemen naar `resolveAndUpdateUserUsing`?